### ISSUE TITLE
camvid weights question

### ISSUE THREAD

**Issue by Jerry2990** [ydhongHIT/DDRNet#39]
hi，thanks for your work and sharing. 
i download the DDRNet_23_slim pre-trained weights on camvid datasets. However, it can't parse images and obtain accurate semantic segmentation results by normalize with `mean_std = ([0.406, 0.456, 0.485], [0.225, 0.224, 0.229])` . Could you tell me how to evaluate with this model. 
Thank you a lot!

PS:
![image](https://github.com/ydhongHIT/DDRNet/assets/50744843/1bc34214-6ae6-402e-8367-e5705540743a)

**Comment by Dileepmeom** [comment]
I also have same issue, output tensor are no where close to the actual prediction. Anyone please suggest how to test the prediction with an actual image instead. Thank you

**Comment by ydhongHIT** [comment]
Hi, my CamVid dataloader is borrowed from DABNet[https://github.com/Reagan1311/DABNet/blob/master/dataset/camvid.py]. Hope this can help you.
